packages/core/utils/src/longRunningProcessTimer.ts

```diff
@@ -1,5 +1,5 @@
 import { red } from "colors";
-import { Logger } from 'embark-logger';
+import { Logger } from 'embark';
```

Comment:
This change and https://github.com/embark-framework/embark/pull/2057/files#diff-fb6fb47a1e3c62a1f838bacba50af184R1 serve the purpose of breaking a circular dependency relationship between `embark-utils` and `embark-logger`.